src/test/java/org/sagebionetworks/bridge/exporter/integration/NotificationTest.java

```diff
@@ -72,6 +73,9 @@
     private static final String STUDY_COMMITMENT_SURVEY_QUESTION = "benefits";
     private static final String STUDY_COMMITMENT_DUMMY_ANSWER = "This is my study commitment";
 
+    // From receivefreesms.com.
+    private static final Phone SECOND_USER_PHONE_NUMBER = new Phone().regionCode("US").number("+14243346702");
```

Comment:
So far as I can tell, anything sent to these phone numbers just gets logged to their web site: http://receivefreesms.com/14243346702.php. Maybe I'm misunderstanding that page but, is that ok with us?

Reply:
I think it's okay if we use it for integ test accounts that are deleted again afterwards. As an alternative, there are services that will provide private test phone numbers, but these generally cost money.

We can discuss more at the team meeting today if we think it's an issue.